py3dtiles/feature_table.py

```
@@ -180,6 +196,9 @@ def from_dtype(positions_dtype, colors_dtype, npoints):
                 fth.positions_dtype = np.dtype([('X', np.uint16),
                                                 ('Y', np.uint16),
                                                 ('Z', np.uint16)])
+                if quantized_volume_offset and quantized_volume_scale:
```

Maybe throw an error if these values are not defined?